### DISCUSSION TITLE
Let's add PipeCD to awesome lists

### DISCUSSION THREAD

**Discussion by khanhtc1202** [pipe-cd/pipecd#4470] [Ideas]
To increase the PipeCD community visibility, one good approach is to add it to awesome lists. In this thread we list all awesome list we know, and let's make PR to add PipeCD there 🌱

**Comment by khanhtc1202** [comment]
https://github.com/collabnix/kubetools - PRed

**Reply by khanhtc1202** [reply]
merged

**Comment by khanhtc1202** [comment]
https://github.com/weaveworks/awesome-gitops - added

**Comment by khanhtc1202** [comment]
https://github.com/rootsongjc/awesome-cloud-native - PRed

**Reply by khanhtc1202** [reply]
merged 🙆‍♀️

**Comment by khanhtc1202** [comment]
https://github.com/wmariuss/awesome-devops - added

**Comment by khanhtc1202** [comment]
https://github.com/avelino/awesome-go

**Reply by khanhtc1202** [reply]
@kentakozuka In order to add the pipecd project to this list, looks like it's required to add codecov badge to the pipecd readme, is that okay? 👀

**Reply by kentakozuka** [reply]
Of course. let's do it 👍

**Reply by khanhtc1202** [reply]
merged 🆗
